models/config.go

```diff
@@ -1,16 +1,79 @@
 package models
 
 import (
+	"context"
+	"database/sql/driver"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"strings"
 	"time"
 
 	"github.com/flanksource/duty/types"
+	"github.com/google/uuid"
 	"github.com/lib/pq"
 	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+	"gorm.io/gorm/schema"
 )
 
+type Config struct {
```

Comment:
Use ConfigItem instead

Reply:
@yashmehrotra Does this mean `models.Config` can be deleted?

Reply:
Yeah, they refer to the same thing

Reply:
okay, deleted it.